content/en/database_as_a_service.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Database as a Service (DBaaS)
+status: Completed
+category: Technology
+---
+
+## What it is
+
+Database-as-a-Service (DBaaS) is a service managed by a cloud operator (public or private) that supports applications without requiring the application team to perform traditional database administration functions. With DBaaS, application developers should not need to be database experts, nor should they need to hire a database administrator (DBA) to keep the database up to date.
+
+## Problem it addresses 
+
+Traditionally in on-premise setups, organizations have to invest in additional storage and processing capacity on a regular basis to accommodate database expansion. Developers also need to provision and configure database with the help of IT infrastructure team. Deployment speed and database driven applications takes longer time to load and execute. This lack of features and scalibilty leads to increased costs and slow applications.
```

Comment:
Please change "on-premise" to "on-premises". 😃

Reply:
Done!!

Reply:
Here also some edits with a few tweaks:

> Traditionally, in on-premise setups, organizations regularly have to invest in additional storage and processing capacity to accommodate database expansion which can be expensive.
> 
> Additionally, developers typically provision and configure databases with the help of IT infrastructure teams, slowing deployment speed of database-driven applications down. Loading and executing them also takes longer.